Framework/Core/src/DataRelayer.cxx

```diff
@@ -335,7 +337,11 @@ void DataRelayer::setOldestPossibleInput(TimesliceId proposed, ChannelIndex chan
         if (element.size() == 0) {
           auto& state = mContext.get<DeviceState>();
           if (state.transitionHandling != TransitionHandlingState::NoTransition && DefaultsHelpers::onlineDeploymentMode()) {
-            LOGP(warning, "Missing {} (lifetime:{}) while dropping incomplete data in slot {} with timestamp {} < {}.", DataSpecUtils::describe(input), input.lifetime, si, timestamp.value, newOldest.timeslice.value);
+            if (state.allowedProcessing == DeviceState::CalibrationOnly) {
```

Comment:
In case we are in calibration mode, I would drop the MISSING message completely. It is expected to be missing, so no need to inform about it.
And then for the "Dropping lifetime::timeframe" warning, I would demote to info in calibration mode and pring " [INFO]: Not processing in calibration mode:...."